library/appointments.inc.php

```diff
@@ -41,7 +41,7 @@
     'trackerstatus' => array( 'trackerstatus', 'date', 'time', 'patient' ),    
 );
 
-function fetchEvents( $from_date, $to_date, $where_param = null, $orderby_param = null, $tracker_board = false, $nextX = 0, $bind_param = null, $query_param = null )
+function fetchEvents( $from_date, $to_date, $where_param = null, $orderby_param = null, $tracker_board = false, $nextX = 0, $bind_param = null, $query_param = null,$facility_name = 0 )
```

Comment:
TODO for me:
Also, if need the $facility_name, then will change it to a boolean.